Location fix

marcvaneijk opened the pull request:
Changed resource location from parameter to resourceGroup().location.
Deleted location parameter from deployment template and parameter file.

singhkay commented:
@marcvaneijk json error `  AssertionError: thinkbox-deadline/metadata.json is not valid JSON. Copy and paste the contents to https://jsonformatter.curiousconcept.com/ and correct the syntax errors. Error: SyntaxError: Unexpected token }`

marcvaneijk commented:
@singhkay 
- fixed language in metadata.json
- storageAccountName from param to var
- replaced disk endpoints to dynamic
- removed storageAccountName from param file
- updated param file with placeholders
